fdm-docs/docs/02-Installation.md

```diff
@@ -0,0 +1,55 @@
+---
+sidebar_position: 2
+title: Installation
+---
+
+Welcome to the Farm Data Model (FDM)! FDM provides a standardized schema and tools for managing and analyzing agricultural data. This guide will walk you through the initial setup and introduce you to the core components of the FDM ecosystem.
+
```

Comment:
add "data on farm and field level" to make it directly clear what the context is. agricultural data is a term also used for FAOSTAT, CBS etc.

Reply:
Resolved in ca64b45ce1b2661d6ab4a85b678cdf61c792b1e9